cobbler/settings.py

```diff
@@ -185,11 +185,14 @@
 class Settings(object):
 
     @staticmethod
-    def collection_type() -> str:
+    def collection_type(self):
         """
-        return the collection name
+        This is a hardcoded string which represents the collection type.
+
+        :return: "settings"
+        :rtype: str
         """
-        return "settings"
+        return "setting"
```

Comment:
@SchoolGuy This is a change to functionality. Has this been checked by anybody?

Reply:
Don't know how this could happen. I will revert this of course.

Reply:
👍